02-technical-guides/atoti-pandas-comparison/README.md

```diff
@@ -0,0 +1,10 @@
+# Atoti pandas Comparison
```

Comment:
@printhellohetal Hmmm not sure if pandas should be capitalized in a title or not according to https://pandas.pydata.org/about/citing.html#:~:text=When%20using%20the%20project%20name,the%20beginning%20of%20a%20sentence. If this is wrong I can amend.